svg/elements/feComposite.json

```diff
@@ -108,10 +108,12 @@
                 "version_added": "≤18"
               },
               "firefox": {
-                "version_added": true
+                "version_added": true,
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://bugzil.la/1171438'>bug 1171438</a>)."
```

Comment:
This is documented in [bug 437554](https://bugzilla.mozilla.org/show_bug.cgi?id=437554) 
- Not yet clear if this also affects the `in` attribute too. 
- [this comment](https://bugzilla.mozilla.org/show_bug.cgi?id=437554#c18) indicates that this also affects other browsers. I'm not sure though. The "workaround" code shows this fails on FF but not the others - run this on both different browsers to see: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/in#result

Reply:
The bug reference seems wrong.

Reply:
@evilpie You're right. Fixed, and rolled out to both in and in2.